### DISCUSSION TITLE
是否兼容任意TCP数据包？

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#2467] [General]
http和https也采用TCP，但是TCP协议能实现的肯定不止http和https。能否转发任意TCP数据包？比如自己随便写一点内容，以TCP形式发出去。

**Comment by mydogshitgold** [comment]
Ja, v2ray can forward arbitrary TCP (and UDP) packets. dokodemo-door port mapping or transparent proxy could be used for this. See[ the official documentation](https://www.v2fly.org/config/overview.html) and [the guide](https://guide.v2fly.org/) for more information.
